bip-update-process.md

```diff
@@ -0,0 +1,648 @@
+licenses *in addition to* at least one acceptable license. In this case, only the acceptable license(s) should be listed
+in the License and License-Code headers.
+
+It is recommended that BIPs that include literal code be dual-licensed under the same license terms as the project it
+modifies. For example, literal code intended for Bitcoin Core would ideally be dual-licensed under the MIT license terms
+as well as one of the above with the rest of the BIP text.
+
+In all cases, details of the licensing terms must be provided in the Copyright section of the BIP.
+
+#### Recommended licenses
+
+* BSD-2-Clause: [OSI-approved BSD 2-clause license](https://opensource.org/licenses/BSD-2-Clause)
+* BSD-3-Clause: [OSI-approved BSD 3-clause license](https://opensource.org/licenses/BSD-3-Clause)
+* CC0-1.0: [Creative Commons CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/)
+* GNU-All-Permissive: [GNU All-Permissive License](http://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html)
+
+#### Not recommended, but acceptable licenses[^licenses]
+
+* CC-BY-4.0: [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/)
+* CC-BY-SA-4.0: [Creative Commons Attribution-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/)
+* MIT: [Expat/MIT/X11 license](https://opensource.org/licenses/MIT)
+
+#### Not acceptable licenses
+
+All licenses not explicitly included in the above lists are not acceptable terms for a Bitcoin Improvement Proposal.
+However, BIPs predating the acceptance of this BIP were allowed under other terms, and should use these abbreviations
+when no other license is granted:
+
+* PD: Released into the public domain
+* OPL: [Open Publication License, version 1.0](http://opencontent.org/openpub/)
+
+## BIP Editors
+
+The current BIP editors are:
+
+* Bryan Bishop ([kanzure@gmail.com](mailto:kanzure@gmail.com))
+* Jon Atack ([jon@atack.com](mailto:jon@atack.com))
+* Luke Dashjr ([luke_bipeditor@dashjr.org](mailto:luke_bipeditor@dashjr.org))
+* Mark "Murch" Erhardt ([murch@murch.one](mailto:murch@murch.one))
+* Olaoluwa Osuntokun ([laolu32@gmail.com](mailto:laolu32@gmail.com))
+* Ruben Somsen ([rsomsen@gmail.com](mailto:rsomsen@gmail.com))
+
+### BIP Editor Responsibilities & Workflow
+
+The BIP editors subscribe to the Bitcoin development mailing list and watch the [BIPs
+repository](https://github.com/bitcoin/bips).
+Off-list BIP-related correspondence should be sent (or CC'd) to the BIP editors.
+
+When a new BIP idea is submitted to the mailing list, BIP editors and other community members should comment in regard
+to:
+
+* Novelty of the idea
+* Viability, utility, and relevance of the concept
+* Readiness of the draft
+* On-topic for the Bitcoin community
+
+We observe that discussion in pull requests comments can often be hard to follow as feedback gets marked as resolved
```

Comment:
```suggestion
We observe that discussion in pull request comments can often be hard to follow as feedback gets marked as resolved
```
Or "pull requests' comments" if you want to refer to plural PRs, but singular seems better to me.